uportal-war/src/main/java/org/jasig/portal/jgroups/protocols/DAO_PING.java

```diff
@@ -134,6 +142,7 @@ protected void getAndSavePhysicalAddress(String clusterName) {
     }
 
     protected void handleView(View view) {
+        log.warn("Change in JGroup membership: " + view.toString());
```

Comment:
Should be INFO, or DEBUG.  You don't want to clutter the logs with these messages, though you hope they are fairly rare and only occur during software updates.